src/create-template/pvs.ts

```diff
@@ -156,22 +158,24 @@ function createItems(
 function createTemplateItem(
   args: CreateTemplateItemArgs
 ): ExtendedTemplateItem {
-  const suppliedId = args.pathId === '' ? '/' : args.pathId;
+  const suppliedId = args.pathId === '' ? PathIdSeparator : args.pathId;
   const parentId =
-    suppliedId === '/'
+    suppliedId === PathIdSeparator
       ? undefined
-      : suppliedId.split('/').slice(0, -1).join('/');
+      : suppliedId.split(PathIdSeparator).slice(0, -1).join(PathIdSeparator);
   const t = args.transform;
 
   return {
+    depth: args.pathId.split(PathIdSeparator).length - 1,
     fileName: args.fileName,
-    name: args.partName,
-    parentId: parentId === '' ? '/' : parentId,
-    source: `/parts?filter[suppliedId]=${encodeURI(
+    suppliedPartId: args.partName,
+    suppliedRevisionId: args.partRevision,
```

Comment:
Pull these into separate fields so I don't have to decode the `source` to get them when creating parts.